package.json

```diff
@@ -104,6 +104,7 @@
     "build:all": "yarn bundle && electron-builder -wl --x64 --ia32 && yarn build:mac && yarn build:snap && yarn build:write-sums",
     "build:mac": "node scripts/build.mjs mac",
     "build:snap": "node scripts/build.mjs snap",
+    "build:deb": "node scripts/build.mjs deb",
```

Comment:
If you want to build a deb and only a deb, you should use `yarn build deb`, which also builds the web app that we depend on 🙂 the wording is a little confusing at the moment, but this line should be removed nonetheless

Reply:
Done!